api4/system_test.go

```diff
@@ -683,11 +683,13 @@ func TestGetServerBusyExpires(t *testing.T) {
 	th.App.Srv().Busy.Set(time.Second * 30)
 
 	t.Run("as system user", func(t *testing.T) {
+		// nolint:staticcheck
```

Comment:
What was the warning here?

Reply:
This is for the deprecated GetServerBusyExpires method, I will Use GetServerBusy.

Reply:
Ah okay, let's keep the `nolint` comment. I'll send a separate PR to remove the method.